app/styles/object_inspector.scss

```diff
@@ -10,3 +10,107 @@
   margin-left: 39px;
   word-break: break-all;
 }
+
+:root {
+  --pink: #909;
+  --green: #318409;
+  --black: #000;
+  --light-black: #333;
+  --white: #fff;
+  --light-pink: #b96aa0;
+}
+
+.icon-service {
+  background-color: #76c76b;
+  border-color: #569c69 !important;
+  &::before {
+    content: "S";
+  }
+}
+
+.icon-computed {
+  background-color: #e358b1;
+  border-color: #740674  !important;
+  &::before {
+    content: "C";
+  }
+}
+
+.mixin__property--group {
+  font-weight: bold;
+}
+
+.mixin__property-dependency-list, .mixin__property-dependency-item {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: block;
+  margin-bottom: 0.2em;
+}
+
+.mixin__property-dependency-item {
+  margin-left: 2.5em;
+  position: relative;
+}
+
+.mixin__property-dependency-item--computed {
+  color: var(--light-pink);
+}
+
+.mixin__property-name--computed {
+  color: var(--pink);
+}
+.mixin__property-name--service {
+  color: var(--green);
+}
+
+.mixin__property-dependency-item::before {
+  content: "";
+  display: block;
+  background: url("data:image/svg+xml;utf8,<svg width='8' height='16' xmlns='http://www.w3.org/2000/svg'><g transform='translate(-14 -16)' stroke='#979797' fill='none' fill-rule='evenodd'><circle cx='18' cy='28' r='3'/><path d='M18 24.571V16.5'/></g></svg>") no-repeat;
```

Comment:
Longer arrow and white circle
```<svg width="8" height="20" xmlns="http://www.w3.org/2000/svg"><g transform="translate(1 -1)" stroke="#979797" fill="none" fill-rule="evenodd"><circle fill="#FFF" cx="3" cy="17" r="3"/><path d="M3 13.571V1.489649"/></g></svg>
```

Reply:
![image](https://user-images.githubusercontent.com/1360552/38601549-a8f65a78-3d70-11e8-9533-81052c2b1d22.png)